From 43d916dc8d0a9f9cff588fd3f68f3da150f09409 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:45:43 +0100
Subject: [PATCH 034/133] std.Target: bump wasi max version to 0.3.0
X-Developer-Signature: v=1; a=openpgp-sha256; l=694; i=xtex@astrafall.org;
 h=from:subject; bh=iU3183h1kzdj/rqozeib1NEjAOk7IMhV4c/Fn4Ah9oE=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv+GLjtndscmzuyawWs191XWB/97qE5Y2eOkMsktn
 89uroxpx0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBU99xh+F/3rseVeWKxwUwp7f6NT+82/1h+7cWdvSvOZLTvviF/8fQuRoYL/sn
 LFHpCNulO4JxrMitTyln6p4Gq1LFeueTVpoxrhfgB
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 27c949689e869289b69a1390706a1cdb04fbad90)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 7f2a7a7a7b40..847a0689e68c 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -608,7 +608,7 @@ pub const Os = struct {
                 .wasi => .{
                     .semver = .{
                         .min = .{ .major = 0, .minor = 1, .patch = 0 },
-                        .max = .{ .major = 0, .minor = 2, .patch = 2 },
+                        .max = .{ .major = 0, .minor = 3, .patch = 0 },
                     },
                 },
 
-- 
2.52.0

